tree/treeplayer/src/TCsvDS.cxx

```diff
@@ -248,16 +248,15 @@ size_t TCsvDS::ParseValue(const std::string &line, std::vector<std::string> &col
 /// \param[in] readHeaders `true` if the CSV file contains headers as first row, `false` otherwise
 ///                        (default `true`).
 /// \param[in] delimiter Delimiter character (default ',').
-TCsvDS::TCsvDS(std::string_view fileName, bool readHeaders, char delimiter) // TODO: Let users specify types?
-   : fFileName(fileName),
-     fDelimiter(delimiter)
+TCsvDS::TCsvDS(std::string_view fileName, bool readHeaders, char delimiter, Long64_t linesChunkSize) // TODO: Let users specify types?
+   : fStream(new std::ifstream(std::string(fileName))),
+     fDelimiter(delimiter), fLinesChunkSize(linesChunkSize)
 {
-   std::ifstream stream(fFileName);
    std::string line;
 
    // Read the headers if present
    if (readHeaders) {
-      if (std::getline(stream, line)) {
+      if (std::getline(*fStream.get(), line)) {
```

Comment:
I think all the `*fStream.get()` could be `*fStream` instead

Reply:
Superseded by the fact it is not a pointer anymore